rfcs/0001-generic-launch-receipting-pattern.md

```diff
@@ -96,6 +96,14 @@ Something to consider is that `survey_metadata.data` can be `survey_metadata` an
 }  
 ```  
 
+#### JWT token length
+
+The platform does not have any specifications/limits on the JWT token length apart from any hard limits imposed by browsers. However, it is essential to understand that this new format would result in a slightly larger token length since the data is provided in a nested structure and could be further increased depending on the size of the `receipting_keys` field.
+
+Although modern browsers have support for much larger URL lengths, the platforms still support Internet Explorer, which has a hard limit of 2,048 characters for the URL path. See: `IE URL Limit <https://support.microsoft.com/en-us/topic/maximum-url-length-is-2-083-characters-in-internet-explorer-174e7c8a-6666-f4e0-6fd6-908b53c12246>`_
```

Comment:
I believe we're about to drop IE11 very soon so this may well be _"the recently unsupported IE"_